source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -63,10 +63,41 @@ level: 4
 ref: install
 content: |
 
-  To install the latest stable version of ``mongosh``, issue the following
-  command:
+  To install the latest stable version of ``mongosh``, issue the
+  following command:
 
   .. code-block:: sh
 
     sudo yum install -y mongodb-mongosh
+  
+  MongoDB also provides a version of ``mongosh`` that uses your system's
+  OpenSSL library.
+  
+  To install ``mongosh`` that uses OpenSSL 1.1 on Red Hat, issue the
+  following command:
+
+  .. code-block:: sh 
+
+     sudo yum install -y mongodb-mongosh-shared-openssl11
+
+  To install ``mongosh`` that uses OpenSSL 1.1 on Ubuntu, issue
+  the following command:
+
+  .. code-block:: sh 
+
+     sudo apt-get install -y mongodb-mongosh-shared-openssl11
+
+  To install ``mongosh`` that uses OpenSSL 3.0 on Red Hat, issue
+  the following command:
+
+  .. code-block:: sh 
+
+     sudo yum install -y mongodb-mongosh-shared-openssl3
+
+  To install ``mongosh`` that uses OpenSSL 3.0 on Ubuntu, issue
+  the following command:
+
+  .. code-block:: sh 
+
+     sudo apt-get install -y mongodb-mongosh-shared-openssl3
```

Comment:
I think there is some confusion here.

```
sudo apt-get install -y mongodb-mongosh-shared-openssl11
sudo apt-get install -y mongodb-mongosh-shared-openssl3
```
is what you do on distributions that use `deb` packages (Ubuntu and Debian) and therefore those instructions should be in the `deb` tab in the page.

```
sudo yum install -y mongodb-mongosh-shared-openssl11
sudo yum install -y mongodb-mongosh-shared-openssl3
```
is what you do on distributions that use `deb` packages (Ubuntu and Debian) and therefore those instructions should be in the `rpm` tab in the page.

For `tar.gz` we need to point people to downloading the `tar.gz` with shared openssl library from the download center and installing that as an alternative to e.g. `mongosh-1.5.0-linux-x64.tgz`.

Reply:
Done.